src/plugin.c

```diff
@@ -288,82 +455,26 @@ int plugin_score(struct qso_t *qso) {
     return result;
 }
 
-bool plugin_is_multi(int band, char *call, int mode) {
-#ifdef HAVE_PYTHON
-    // call is_multi
-    struct qso_t qso;
-    qso.band = band;
-    qso.call = call;
-    qso.mode = mode;
-    qso.comment = "";
-
-    PyObject *py_qso = create_py_qso(&qso);
-    PyObject *args = Py_BuildValue("(O)", py_qso);
-    PyObject *pValue = PyObject_CallObject(pf_is_multi, args);
-    Py_DECREF(args);
-    Py_DECREF(py_qso);
-
-    bool result = false;
-    if (pValue != NULL) {
-	result =  PyLong_AsLong(pValue);
-    }
-    Py_XDECREF(pValue);
-
-    if (NULL != PyErr_Occurred()) {
-	PyErr_Print();
-	sleep(2);
-	//exit(1);
-    }
-    return result;
-#else
-    return false;
-#endif
-}
-
-// result has to be g_freed()'s
-char *plugin_get_multi(struct qso_t *qso) {
+void plugin_check_exchange(struct qso_t *qso) {
```

Comment:
Seems to me that counting of the multipliers is not correct.

On last contest of MWC I got 42 multipliers:
```
$ grep " [A-Z] " ok1wc.log  | wc -l
42
```
but Tlf shows 61 multipliers:

![Képernyőkép 2023-03-04 19-38-45](https://user-images.githubusercontent.com/238977/222923190-ea4a9220-22ea-49c2-bdc2-0de4c4da949d.png)

Reply:
In the 'plugin_cleanup' branch is a fix missing. See commit 17070eaabf1060a19856f0c6ba809e86b48ec595 . Just cherry-picking it to the plugin_branch should fix the problem.

Reply:
> In the 'plugin_cleanup' branch is a fix missing.

Thanks Thomas, I can confirm that picking that commit solved the multiplier issue.

Reply:
Also for the multipliers: this hack modifies the behavior of the logger. Tlf shows the multipliers per band in the `Mult` line:

```diff
$ git diff src/showscore.c
diff --git a/src/showscore.c b/src/showscore.c
index 6a14c76..059e087 100644
--- a/src/showscore.c
+++ b/src/showscore.c
@@ -245,7 +245,8 @@ void showscore(void) {
     if (wysiwyg_multi
            || serial_section_mult
            || serial_grid4_mult
-           || sectn_mult) {
+           || sectn_mult
+           || strlen(plugin_config) > 0) {
 
        mvaddstr(3, START_COL, "Mult ");
        for (i = 0; i < 6; i++) {
```

Now should I sent it to the Zoli's branch, or finish this plugin stage first, and later continue the work in the master branch?

Reply:
I just merged actual master into plugin_cleanup branch and updated the branch on https://github.com/Tlf/tlf. 
The multiplier display is yet to be done.